utils/auth.ts

```diff
@@ -0,0 +1,52 @@
+const AUTH_SESSION_KEY = 'likecoin_nft_book_press_token'
+const POST_AUTH_REDIRECT_KEY = 'likecoin_nft_book_press_post_auth_redirect'
+
+export function loadAuthSession () {
+  try {
+    const data = window.localStorage.getItem(AUTH_SESSION_KEY)
```

Comment:
Ideally we should check for local/session storage function exists before using them in this file